packages/core/tsconfig.cjs.json

```diff
@@ -0,0 +1,12 @@
+// Note: Because we can't inherit from more than one `tsconfig` file, every package must have its own
+// `tsconfig.cjs.json` with the `outDir` and `module` options below
```

Comment:
Feel like we don't need to repeat this note in every file, can we just lint for this instead?

Reply:
I'm seeing it more as boilerplate explanation than an attempt to make sure it happens by saying it should. I was just imagining someone coming to the code, seeing the same file in all sorts of packages, and wondering why we didn't just pull it out into a single `tsconfig.esm.json` at the repo level. But I also can just leave it in the template and/or the readme if you think it's overkill. Thoughts?